{"name": "FireBase_Verify scan details API returns 400 for invalid scan ID format -- @1.1 ", "status": "broken", "statusDetails": {"message": "requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://identitytoolkit.googleapis.com/v1/accounts:signInWithPassword?key=AIzaSyD08yxn7V_YAfkAY8mKDTYjfKV_qoPrdRQ\n", "trace": "  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\.venv\\Lib\\site-packages\\behave\\model.py\", line 1991, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\.venv\\Lib\\site-packages\\behave\\matchers.py\", line 105, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\FireBase_get_a_specific_sca_by_id.py\", line 22, in step_impl\n    id_token = get_firebase_id_token()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\utilities\\authentication.py\", line 16, in get_firebase_id_token\n    response.raise_for_status()  # Raise an error for non-2xx responses\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\.venv\\Lib\\site-packages\\requests\\models.py\", line 1026, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\n"}, "steps": [{"name": "Given FireBase_User sends GET request to fetch scan details for scan ID \"invalid\"", "status": "broken", "statusDetails": {"message": "requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://identitytoolkit.googleapis.com/v1/accounts:signInWithPassword?key=AIzaSyD08yxn7V_YAfkAY8mKDTYjfKV_qoPrdRQ\n", "trace": "  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\.venv\\Lib\\site-packages\\behave\\model.py\", line 1991, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\.venv\\Lib\\site-packages\\behave\\matchers.py\", line 105, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\FireBase_get_a_specific_sca_by_id.py\", line 22, in step_impl\n    id_token = get_firebase_id_token()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\utilities\\authentication.py\", line 16, in get_firebase_id_token\n    response.raise_for_status()  # Raise an error for non-2xx responses\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\.venv\\Lib\\site-packages\\requests\\models.py\", line 1026, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\n"}, "start": 1757693669851, "stop": 1757693670450}, {"name": "When FireBase_the user receives the scan details response", "status": "skipped", "start": 1757693670455, "stop": 1757693670455}, {"name": "Then FireBase_the status code returned should be 400 for bad request error", "status": "skipped", "start": 1757693670455, "stop": 1757693670455}, {"name": "And FireBase_the response should contain an error message \"Bad Request\"", "status": "skipped", "start": 1757693670455, "stop": 1757693670455}, {"name": "And FireBase_the message should contain \"Invalid input:\"", "status": "skipped", "start": 1757693670455, "stop": 1757693670455}], "parameters": [{"name": "scan_id", "value": "invalid"}], "start": 1757693669848, "stop": 1757693670456, "uuid": "63e33887-ce58-4b8a-a9a4-61c16e226d2d", "historyId": "fddca657e47f241692da66c3d13bd0c2", "fullName": "FireBase_Get Scan Details using Genr3d API: FireBase_Verify scan details API returns 400 for invalid scan ID format", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "Negative"}, {"name": "tag", "value": "Security"}, {"name": "tag", "value": "Test"}, {"name": "feature", "value": "FireBase_Get Scan Details using Genr3d API"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}], "titlePath": ["features", "FireBase_Get Scan Details using Genr3d API"]}